### PR TITLE
pkg/kubelet: minor fixes for cadvisor.go

### PR DESCRIPTION
**Pull request by xiang90**

### CODE REVIEW

**Comment by xiang90** [comment]
Is there any way for me to get rid of the CLA checking? I am using my CoreOS email in my git config.

**Comment by vmarmol** [comment]
Not as far as I know :( we're working on improving that

**Comment by xiang90** [comment]
@vmarmol The cute bot just went against you!

**Comment by vmarmol** [comment]
I want to say it is the first time in my life that a robot has undermined me, but that would be a lie :P

File: pkg/kubelet/cadvisor.go [hunk 1]
```diff
@@ -69,7 +69,7 @@ func (kl *Kubelet) statsFromDockerContainer(cc cadvisorInterface, containerId st
 func (kl *Kubelet) GetContainerInfo(podFullName string, uid types.UID, containerName string, req *cadvisor.ContainerInfoRequest) (*cadvisor.ContainerInfo, error) {
 	cc := kl.GetCadvisorClient()
 	if cc == nil {
-		return nil, nil
+		return nil, fmt.Errorf("no cadvisor connection")
```
Comment (vmarmol) [inline pkg/kubelet/cadvisor.go]: Nice catch :)

**Comment by vmarmol** [comment]
LGTM, thanks @xiang90!

**Pull request merged**